### PR TITLE
Create working group payment poll

### PR DESCRIPTION
**Pull request by monet-supply**
No addresses included yet for other working group members. Wanted to submit to see if the rest of the poll document looks good. Thanks!

### CODE REVIEW

File: governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Vault Compensation Working Group Payment - January 11, 2021
+summary: Signal your support or opposition to paying 12,700 Dai from the surplus buffer to the vault compensation working group for past labor.
+discussion_link: https://forum.makerdao.com/t/signal-request-vault-compensation-working-group-payment/5562
+vote_type: Plurality Voting
+options:
```
Comment (LongForWisdom) [inline governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md:6]: We have categories now:
`categories:`
`   - Other`
See Inclusion Poll - $mip_code - $date_MONTH_DD,_YYYY.md template for syntax if you have trouble. Feel free to ignore this and I'll fix it later.

File: governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Vault Compensation Working Group Payment - January 11, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Governance Community. The community can vote in this poll to express support or opposition to transferring 12,700 Dai from the surplus buffer to members of the vault compensation working group for their past work organizing the vault compensation proposals. Funds will be allocated to individual members as follows:
```
Comment (LongForWisdom) [inline governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md:13]: So I've started moving poll-specific detail to the review section. The only info under the top-level heading should be the meta-detail of the poll itself (ie when it is, how it got up on the portal, type of poll, etc.)

MIP9 Community Greenlight Poll - WFIL - December 14, 2020.md for an example.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Looks good @monet-supply, just some minor changes. Remember to update all the dates if we move this to next week.

File: governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md (hunk 2 above)
Comment (monet-supply) [reply governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md:13]: Awesome, thank you!

Comment (monet-supply) [reply governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md]: updated in commit 63b048f0a5f8a741c627b2896faae94125c222b6

File: governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md (hunk 1 above)
Comment (monet-supply) [reply governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md:6]: thanks

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**